step.yml

```diff
@@ -1,15 +1,12 @@
 title: "Project scanner"
-summary: Scans repository for iOS, macOS, Android, Xamarin, Fastlane and Cordova projects
+summary: Scans repository for iOS, macOS, Android, Fastlane and Cordova projects
```

Comment:
NOTE: some supported project types are missing here

Reply:
I have added all of the missing project types.